packages/belos/tpetra/test/TFQMR/test_tfqmr_hb.cpp

```diff
@@ -94,46 +110,43 @@ int main(int argc, char *argv[]) {
     int numrhs = 1;  // total number of right-hand sides to solve for
     int maxiters = -1;  // maximum number of iterations for solver to use
     std::string filename("orsirr1.hb");
-    double tol = 1.0e-5;  // relative residual tolerance
-
+    ST tol = sqrt(std::numeric_limits<ST>::epsilon()); // relative residual tolerance
```

Comment:
See [this comment](https://github.com/NexGenAnalytics/Trilinos/pull/76#pullrequestreview-1613935197) from the Belos package manager. I ended up going back to hard-coding the tolerance for `double`, since `float` doesn't work for now.

Reply:
Thanks for this comment! Fixed now